python/TestHarness/testers/RunApp.py

```diff
@@ -114,7 +114,9 @@ def getCommand(self, options):
       specs['cli_args'].append('Outputs/print_perf_log=true')
 
     if options.colored == False:
-      specs['cli_args'].append('--no-color')
+      specs['cli_args'].append('--color off')
+    else:
+      specs['cli_args'].append('--color on')
```

Comment:
Do we need the condition in the else clause? Shouldn't the default be "auto" in the case that no flag is passed through the `TestHarness`?